8-gen-2012 13.50.22 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:124)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
8-gen-2012 13.56.45 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:124)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
8-gen-2012 14.17.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.renameBuddy(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.renameBuddy(SkypeBuddyRoster.java:169)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testRenameBuddy(SkypeBuddyRosterTest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:24)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
8-gen-2012 14.17.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:124)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:24)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
8-gen-2012 14.20.05 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:124)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.24.45 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.24.49 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.removeGroup(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.removeGroup(SkypeBuddyRoster.java:186)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testRemoveGroup(SkypeBuddyRosterTest.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.26.37 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:120)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.26.38 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.removeGroup(..)]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.removeGroup(SkypeBuddyRoster.java:188)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testRemoveGroup(SkypeBuddyRosterTest.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.28.35 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.28.38 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.removeGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.GetCustomContactGroups(Skype.java:616)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.removeGroup(SkypeBuddyRoster.java:186)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testRemoveGroup(SkypeBuddyRosterTest.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.42.17 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-gen-2012 12.44.05 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
16-gen-2012 18.35.21 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.refreshBuddies()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.refreshBuddies(SkypeBuddyRoster.java:67)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.reload(SkypeBuddyRoster.java:218)
	at it.uniba.di.cdg.skype.SkypeBackend.OnPropertyChange(SkypeBackend.java:706)
	at it.uniba.di.cdg.skype.SkypeListeners.OnPropertyChange(SkypeListeners.java:215)
	at com.skype.api.Skype.HandlePropertyChange(Skype.java:176)
	at com.skype.ipc.EventDispatcher.run(EventDispatcher.java:92)
16-gen-2012 19.25.31 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.refreshBuddies()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.refreshBuddies(SkypeBuddyRoster.java:67)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.reload(SkypeBuddyRoster.java:218)
	at it.uniba.di.cdg.skype.SkypeBackend.OnPropertyChange(SkypeBackend.java:706)
	at it.uniba.di.cdg.skype.SkypeListeners.OnPropertyChange(SkypeListeners.java:215)
	at com.skype.api.Skype.HandlePropertyChange(Skype.java:176)
	at com.skype.ipc.EventDispatcher.run(EventDispatcher.java:92)
16-gen-2012 20.18.16 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.refreshBuddies()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.refreshBuddies(SkypeBuddyRoster.java:67)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.reload(SkypeBuddyRoster.java:218)
	at it.uniba.di.cdg.skype.SkypeBackend.OnPropertyChange(SkypeBackend.java:706)
	at it.uniba.di.cdg.skype.SkypeListeners.OnPropertyChange(SkypeListeners.java:215)
	at com.skype.api.Skype.HandlePropertyChange(Skype.java:176)
	at com.skype.ipc.EventDispatcher.run(EventDispatcher.java:92)
16-gen-2012 20.18.16 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.refreshBuddies()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.refreshBuddies(SkypeBuddyRoster.java:67)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.reload(SkypeBuddyRoster.java:218)
	at it.uniba.di.cdg.skype.SkypeBackend.OnPropertyChange(SkypeBackend.java:706)
	at it.uniba.di.cdg.skype.SkypeListeners.OnPropertyChange(SkypeListeners.java:215)
	at com.skype.api.Skype.HandlePropertyChange(Skype.java:176)
	at com.skype.ipc.EventDispatcher.run(EventDispatcher.java:92)
9-feb-2012 16.28.05 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.28.06 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.28.06 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.28.07 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.46.43 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeConnect()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:415)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:50)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.50.17 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeConnect()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:415)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:49)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.51.29 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeConnect()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:415)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:49)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.pde.internal.junit.runtime.RemotePluginTestRunner.main(RemotePluginTestRunner.java:62)
	at org.eclipse.pde.internal.junit.runtime.PlatformUITestHarness$1.run(PlatformUITestHarness.java:47)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4140)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3757)
	at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2696)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2660)
	at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2494)
	at org.eclipse.ui.internal.Workbench$7.run(Workbench.java:674)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:667)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:123)
	at org.eclipse.pde.internal.junit.runtime.NonUIThreadTestApplication.runApp(NonUIThreadTestApplication.java:54)
	at org.eclipse.pde.internal.junit.runtime.UITestApplication.runApp(UITestApplication.java:41)
	at org.eclipse.pde.internal.junit.runtime.NonUIThreadTestApplication.start(NonUIThreadTestApplication.java:48)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
9-feb-2012 16.53.48 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeConnect()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:415)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:47)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.54.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.login(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.GetAccount(Skype.java:2415)
	at it.uniba.di.cdg.skype.SkypeBackend.login(SkypeBackend.java:673)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:456)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:47)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.56.58 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.login(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.GetAccount(Skype.java:2415)
	at it.uniba.di.cdg.skype.SkypeBackend.login(SkypeBackend.java:673)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:455)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:47)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.58.38 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.login(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.GetAccount(Skype.java:2415)
	at it.uniba.di.cdg.skype.SkypeBackend.login(SkypeBackend.java:673)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:455)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:47)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.pde.internal.junit.runtime.RemotePluginTestRunner.main(RemotePluginTestRunner.java:62)
	at org.eclipse.pde.internal.junit.runtime.PlatformUITestHarness$1.run(PlatformUITestHarness.java:47)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4140)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3757)
	at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2696)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2660)
	at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2494)
	at org.eclipse.ui.internal.Workbench$7.run(Workbench.java:674)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:667)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:123)
	at org.eclipse.pde.internal.junit.runtime.NonUIThreadTestApplication.runApp(NonUIThreadTestApplication.java:54)
	at org.eclipse.pde.internal.junit.runtime.UITestApplication.runApp(UITestApplication.java:41)
	at org.eclipse.pde.internal.junit.runtime.NonUIThreadTestApplication.start(NonUIThreadTestApplication.java:48)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
9-feb-2012 17.06.45 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.login(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.GetAccount(Skype.java:2415)
	at it.uniba.di.cdg.skype.SkypeBackend.login(SkypeBackend.java:673)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:456)
	at it.uniba.di.cdg.skype.test.SkypeBackendStateTest.setUp(SkypeBackendStateTest.java:50)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 17.11.50 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 17.22.19 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:24)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 20.35.30 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
10-feb-2012 12.30.13 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.addGroup(..)]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at com.skype.ipc.RootObject.notConnected(RootObject.java:218)
	at com.skype.ipc.RootObject.XCall(RootObject.java:165)
	at com.skype.api.Skype.CreateCustomContactGroup(Skype.java:651)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.addGroup(SkypeBuddyRoster.java:118)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.testAddContainsBuddy(SkypeBuddyRosterTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
